frontend/src/scenes/dashboard/DashboardItem.js

```diff
@@ -122,6 +137,10 @@ export function DashboardItem({
                         </Link>
                     </div>
                     <div className="dashboard-item-settings">
+                        <span style={{ cursor: 'pointer', marginTop: -3 }} onClick={() => loadResults(true)}>
+                            {item.last_refresh && <i> Updated: {moment(item.last_refresh).fromNow()}</i>}
```

Comment:
The word "updated" leads me to believe that someone else changed something about the graph. Maybe "refreshed" is better.

From an aesthetic point of view that whole sentence takes up a lot of width and space. I actually don't think we need to show this at all. We could just put it in the tooltip over `ReloadOutlined`